ratpack-bearer-auth/src/main/java/st/ratpack/auth/ValidateTokenResult.java

```diff
@@ -0,0 +1,48 @@
+package st.ratpack.auth;
+
+public interface ValidateTokenResult {
+
+	Status getStatus();
+
+	OAuthToken getOAuthToken();
+
+	default Boolean isErrorResult() {
+		return this.getStatus().equals(Status.UNKNOWN);
```

Comment:
Could we switch the order of these to ensure no NPEs?  
i.e.
```
return Status.UNKNOWN.equals(this.getStatus())
```

Reply:
ha yeah good catch!